### ISSUE TITLE
NGLView visualisations are broken in the published documentation

### ISSUE THREAD

**Issue by pgrigorev** [libAtoms/matscipy#261]
The current version of published documentation shows JavaScript error in the place on NGLView visualisations.

```
[Open Browser Console for more detailed log - Double click to close this message]
Failed to load model class 'NGLModel' from module 'nglview-js-widgets'
ReferenceError: s is not defined
    at Object. (https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:12109:4412)
    at __webpack_require__ (https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:20:30)
    at Object. (https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:69164:11)
    at __webpack_require__ (https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:20:30)
    at Object. (https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:69141:18)
    at __webpack_require__ (https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:20:30)
    at https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:63:18
    at https://cdn.jsdelivr.net/npm/nglview-js-widgets@3.1.0/dist/index.js:66:10
    at Object.execCb (https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js:1:12861)
    at b.check (https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js:1:6577)
```

I am not sure if that is similar to what was happening before. 
I can also see a lot of warnings (>1000) in the build log of the docs.

**Comment by pgrigorev** [comment]
At the end it is the same issue as #222 it is just that the version was changed by [this sneaky commit from `renovate[bot]`](https://github.com/libAtoms/matscipy/commit/6b429ce28bbe1b4e02b00689314495881ef40fce). I will prepare a MR to fix it.

**Comment by pastewka** [comment]
Fixed by #262
